nox/virtualenv.py

```diff
@@ -67,6 +70,46 @@ def _clean_location(self):
         return True
 
     @property
+    def _resolved_interpreter(self):
+        """Return the interpreter, appropriately resolved for the platform.
+
+        Based heavily on tox's implementation (tox/interpreters.py).
+        """
+        # Sanity check: If there is no assigned interpreter, then
+        # do nothing.
+        if not self.interpreter:
+            return self.interpreter
+
+        # Sanity check: We only need special behavior on Windows.
+        if platform.system() != 'Windows':
+            return self.interpreter
+
+        # We may have gotten a fully-qualified intepreter path (for someone
+        # _only_ testing on Windows); accept this.
+        if py.path.local.sysfind(self.interpreter):
+            return self.interpreter
+
+        # If this is a standard Unix "pythonX.Y" name, it should be found
+        # in a standard location in Windows.
+        match = re.match(r'^python(?P<maj>\d)\.(?P<min>\d)$', self.interpreter)
+        if match:
+            version = match.groupdict()
+            potential_paths = (
+                r'c:\python{maj}{min}\python.exe'.format(**version),
+                r'c:\python{maj}{min}-x64\python.exe'.format(**version),
+            )
+            for path in potential_paths:
+                actual = py.path.local(path)
+                if actual.check():
```

Comment:
Previously I was returning `str(actual)`, but I changed that to returning path because of details in how the unit tests trick the system.